composer.json

```diff
@@ -9,18 +9,30 @@
         }
     ],
     "require": {
-        "jms/metadata": "~1.5",
-        "symfony/property-access": "~2.7 || ~3.0 || ~4.0",
-        "php": ">=5.4.0"
+        "jms/metadata": "^2.0",
+        "symfony/console": "~3.4 || ~4.0",
+        "symfony/event-dispatcher": "~3.4 || ~4.0",
+        "symfony/expression-language": "~3.4 || ~4.0",
+        "symfony/dependency-injection": "~3.4 || ~4.0",
+        "symfony/property-access": "~3.4 || ~4.0",
+        "php": ">=7.2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8 || ^5.7",
+        "phpunit/phpunit": "^5.7.27",
+        "phpspec/prophecy": "^1.7",
 
         "zendframework/zend-stdlib": "2.3.1 as 2.0.0rc5",
         "zendframework/zendsearch": "2.*@dev",
-        "elasticsearch/elasticsearch": "^2.1",
+        "elasticsearch/elasticsearch": "^2.1 || ^5.0",
         "symfony-cmf/testing": "^1.3 || ^2.1",
-        "symfony/symfony": "~2.7 || ~3.0 || ~4.0",
+        "symfony/filesystem": "~3.4 || ~4.0",
+        "symfony/finder": "~3.4 || ~4.0",
+        "symfony/http-kernel": "~3.4 || ~4.0",
+        "symfony/framework-bundle": "~3.4 || ~4.0",
+        "symfony/form": "~3.4 || ~4.0",
+        "symfony/monolog-bridge": "~3.4 || ~4.0",
+        "symfony/validator": "~3.4 || ~4.0",
+        "symfony/templating": "~3.4 || ~4.0",
```

Comment:
Since we tag that as 2.0, shouldn't we also support only the latest symfony version here?

Reply:
Will upgrade the dependencies.